packages/rxtranscribe/src/operators/toAWS.test.js

```diff
@@ -0,0 +1,88 @@
+import fs from 'fs';
+import path from 'path';
+import {expect} from 'chai';
+import sinon from 'sinon';
+import {from,Observable,of} from 'rxjs';
+import {map,mapTo,take,tap} from 'rxjs/operators';
+import {marbles} from 'rxjs-marbles/mocha';
```

Comment:
I use marble testing frequently.  If you haven't used it before, [RxJS has docs](https://rxjs.dev/guide/testing/marble-testing) describing the syntax/shorthand.  [rxjs-marbles](https://github.com/cartant/rxjs-marbles) provides some syntactic sugar that makes marble testing more concise in mocha tests.